_source/_posts/2022-08-12-ci-cd-microservices-spinnaker.md

```diff
@@ -0,0 +1,952 @@
+---
+layout: blog_post
+title: "Continous Integration and Delivery for JHipster Microservices"
```

Comment:
```suggestion
title: "CI/CD for JHipster Microservices with CircleCI and Spinnaker"
```

I searched for good keywords on Keywork Planner and couldn't find any. Therefore, I think we should just use the key technologies and hope the community benefits more than search engines. ;)